icubam/db/synchronizer.py

```diff
@@ -8,6 +8,11 @@
 
 ICU_COLUMNS = ['name', 'region', 'dept', 'city', 'lat', 'long', 'telephone']
 USER_COLUMNS = ['icu_name', 'name', 'telephone', 'description']
+BC_COLUMNS = [
```

Comment:
btw you can get those with `store.BedCount.get_column_names(include_relationship=False)`
Same above

Reply:
So that is the full set, but since I use this to test that the provide dataframe has the correct columns, I want only a subset (the necessary columns).... So either I use the ones from the store and explicitly delete unnecessary columns, let's discuss offline.